frontend/src/pages/admin/user_group_table/index.tsx

```diff
@@ -66,9 +73,15 @@ export default (props: {
       </div>
       <Table className={cx('table')} columns={columns}>
         {wiredUserGroups.render(data => <>
-          {data?.map(group => <TableRow key={group.slug} data={rowBuilder(group, usersInGroup(wiredUserGroups, group), modifyActions(group, setDeletingUserGroup, setModifyingUserGroup))} />)}
+          {data?.map((group, i) => {
+            const belowUpperBound = i < page * itemsPerPage 
+            const aboveLowerBound = i >= (page - 1) * itemsPerPage
+            const inPageRange = belowUpperBound && aboveLowerBound
+            return inPageRange && <TableRow key={group.slug} data={rowBuilder(group, usersInGroup(wiredUserGroups, group), modifyActions(group, setDeletingUserGroup, setModifyingUserGroup))} />
+          })}
         </>)}
       </Table>
+      <StandardPager className={cx('user-table-pager')} page={page} maxPages={pageLength} onPageChange={(newPage) => setPage(newPage)} />
```

Comment:
Looks good, but you might  be able to use `onPageChange={setPage}`. I'm good either way.

Reply:
good point! Just changed it